src/tallies/filter_mesh.cpp

```diff
@@ -66,6 +90,18 @@ MeshFilter::set_mesh(int32_t mesh)
   n_bins_ = model::meshes[mesh_]->n_bins();
 }
 
+void MeshFilter::set_translation(const Position& translation)
+{
+  translated_ = true;
+  translation_ = translation;
+}
+
+void MeshFilter::set_translation(const double translation[3])
+{
+  translated_ = true;
+  translation_ = translation;
+}
```

Comment:
I would have this implementation use the `set_translation(const Position&)` version

Reply:
I ended up removing this as a parameter of the class and made it an attribute instead.